Makefile

```diff
@@ -45,9 +45,9 @@ else
 	BUILD_CLIENT = false
 endif
 
-# Golang Flags
-GOPATH ?= $(shell go env GOPATH)
+# Go Flags
 GOFLAGS ?= $(GOFLAGS:) -mod=vendor
+GOBIN=$(PWD)/bin
```

Comment:
What is the reasoning behind not use the default install location at `$HOME/go/bin`? Version pinning?

Reply:
Yes, partly that, but mainly not polluting the global namespace. Folks might have other projects using mockery at some other versions. We do not want to clash with them. Hence it is best to keep a project's tools localized to that project.